src/components/audio-feedback.js

```diff
@@ -20,60 +20,38 @@ AFRAME.registerComponent("networked-audio-analyser", {
       sum += this.levels[i];
     }
     this.volume = sum / this.levels.length;
-    this.el.emit("audioFrequencyChange", {
-      volume: this.volume,
-      levels: this.levels
-    });
   }
 });
 
 AFRAME.registerComponent("matcolor-audio-feedback", {
-  schema: {
-    analyserSrc: { type: "selector" }
-  },
-  init: function() {
-    this.onAudioFrequencyChange = this.onAudioFrequencyChange.bind(this);
-  },
+  tick() {
+    if (!this.mat) return;
 
-  play() {
-    (this.data.analyserSrc || this.el).addEventListener("audioFrequencyChange", this.onAudioFrequencyChange);
-  },
+    const audioAnalyser = this.el.components["networked-audio-analyser"];
```

Comment:
Probably not a big deal either way, but maybe it makes more sense to store a refference to this in `init()` or `play()`?

Reply:
I think this is fine.